src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  flex-wrap: wrap;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &__picture {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-title {
```

Comment:
```suggestion
  &__product {
    &-title {
      &:hover {}
    }
    &-code {}
```

Reply:
then the change of color will apply only to the title itself while hovering it. but the task says it should be applied while hovering card